api/handler/attributes.go

```diff
@@ -132,6 +140,21 @@ func parseGetObjectAttributeArgs(r *http.Request) (*GetObjectAttributesArgs, err
 		res.Attributes = append(res.Attributes, a)
 	}
 
+	var err error
+	maxPartsVal := r.Header.Get(api.AmzMaxParts)
+	if maxPartsVal == "" {
+		res.MaxParts = layer.MaxSizePartsList
+	} else if res.MaxParts, err = strconv.Atoi(maxPartsVal); err != nil || res.MaxParts < 0 {
+		return nil, errors.GetAPIError(errors.ErrInvalidMaxKeys)
+	}
+
+	markerVal := r.Header.Get(api.AmzPartNumberMarker)
+	if markerVal == "" {
+		res.PartNumberMarker = partNumberMarkerDefault
```

Comment:
Doesn't `partNumberMarkerDefault` look excessive?  Because there is no difference between marker -1 and 0.